datasets/opus_gnome/README.md

```diff
@@ -41,7 +41,6 @@ language:
 - fa
 - fi
 - fo
-- foo
```

Comment:
Thanks for removing this tag. I searched for it but I was not able to find any reference to it. Looking at the samples in opus_gnome, it seems just English. I'm wondering where this `foo` tag comes from...

If we cannot find any external reference to it, I agree it is better to remove it.